playwright/pages/basePage.ts

```diff
@@ -313,7 +313,7 @@ export class BasePage {
 
   async waitForDOMToFullyLoad() {
     await this.page.waitForLoadState("domcontentloaded", { timeout: 70000 });
-    await this.loader.waitFor({ state: "hidden" });
+    await this.dataGridLoader.waitFor({ state: "hidden" });
```

Comment:
question: this function is used in many places not always with data grid. I'd suggest wait here for previous loader location